agents/data-science/data_science/__init__.py

```
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
```

Ruff (0.8.2)

`os` imported but unused

(F401)